src/accordion/accordion.ts

```diff
@@ -94,8 +125,10 @@ export interface NgbPanelChangeEvent {
   <div class="card">
     <template ngFor let-panel [ngForOf]="panels">
       <div [class]="'card-header ' + (panel.type ? 'card-'+panel.type: type ? 'card-'+type : '')" [class.active]="isOpen(panel.id)">
-        <a tabindex="0" href (click)="!!toggle(panel.id)" [class.text-muted]="panel.disabled">
-          {{panel.title}}<template [ngTemplateOutlet]="panel.titleTpl?.templateRef"></template>          
+        <template *ngIf="panel.headerTpl" [ngTemplateOutlet]="panel.headerTpl?.templateRef" [ngOutletContext]="getHeaderContext(panel)">
+        </template>
+        <a *ngIf="!panel.headerTpl" tabindex="0" href (click)="!!toggle(panel.id)" [class.text-muted]="panel.disabled">
```

Comment:
Indeed. I went back and forth between exposing the NgbPanel itself in the context, and not doing it. It's only useful for this `disabled` flag, but the disabled flag itself is normally coming from the model, which is accessible from the template. So I finally decided to not expose the panel in the context.